scripts/owned_workstation_capability.py

```
@@ -158,6 +262,22 @@ def build_snapshot(
+        "repository": "teamleaderleo/glaeda",
+        "source": {"commitOid": commit_oid, "treeOid": tree_oid},
+        "sourceObjectClass": "exact_commit_and_tree_present",
+        "verificationProfiles": sorted(
+            set(verification_profiles) | set(verification_profile_generations)
+        ),
+    }, *repo_query_projects]
+    if len(projects) > MAX_PROJECTS:
+        raise SnapshotError("Owned-workstation capability has too many projects")
+    repositories = [project.get("repository") for project in projects]
+    if len(repositories) != len(set(repositories)):
+        raise SnapshotError("Owned-workstation capability has duplicate projects")
```

Ruff (0.16.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)